addons/account/models/account_journal_dashboard.py

```diff
@@ -78,12 +78,13 @@ def _graph_title_and_key(self):
     # Below method is used to get data of bank and cash statemens
     def get_line_graph_datas(self):
         """Computes the data used to display the graph for bank and cash journals in the accounting dashboard"""
+        currency = self.currency_id or self.company_id.currency_id
 
         def build_graph_data(date, amount):
             #display date in locale format
             name = format_date(date, 'd LLLL Y', locale=locale)
             short_name = format_date(date, 'd MMM', locale=locale)
-            return {'x':short_name,'y': amount, 'name':name}
+            return {'x':short_name,'y': "%.2f" % amount, 'name':name}
```

Comment:
You could call `currency.round` at each call of `build_graph_data` instead, e.g. `data.append(build_graph_data(today, currency.round(last_balance)))`